### PR TITLE
ADD communication links

### PR DESCRIPTION
**Pull request by robnester-rh**
Add links to issues and discussions under `Contributing` section

resolves: EC-209

### CODE REVIEW

File: website/layouts/index.html [hunk 1]
```diff
@@ -72,7 +72,9 @@ <h2>Recently Published</h2>
           <h2>Collaboration is Welcomed!</h2>
           <p><img class="icon" src="/images/collaboration.svg"/></p>
           <p>We want to invite everyone to help to guide the future of Enterprise Contract.</p>
-          <p>Find our <a href="https://github.com/enterprise-contract/" title="View the Enterprise Contract organization on GitHub">code on GitHub</a>.</p>
+          <p>View the <a href="https://github.com/enterprise-contract/ec-cli" title="View the Enterprise Contract organization on GitHub">code on GitHub</a></p>
+          <p>Found a bug with enterprise contract? Open an <a href="">issue</a></p>
```
Comment (lcarva) [inline website/layouts/index.html:76]: Empty link?

File: website/layouts/index.html [hunk 2]
```diff
@@ -72,7 +72,9 @@ <h2>Recently Published</h2>
...
+          <p>Have questions? Enhancement ideas? Join the <a href="https://github.com/enterprise-contract/ec-cli/discussions">discussion</a>!</p>
```
Comment (lcarva) [inline website/layouts/index.html:77]: What if it's not ec-cli related?

**Comment by lcarva** [comment]
I think we need to be a bit  more verbose in how we define the communication norms. The story calls for a dedicated page for this reason, but a dedicated page is definitely not required IMO (but it might be nice as we can share a direct link!)

**Comment by robnester-rh** [comment]
> I think we need to be a bit more verbose in how we define the communication norms. The story calls for a dedicated page for this reason, but a dedicated page is definitely not required IMO (but it might be nice as we can share a direct link!)

After looking at this story, I didn't think we really needed a whole page to direct folks to discussions and issues. I inadvertently pushed without the issue link however. I'll fix that and look at perhaps pulling this into a separate page as well.

File: website/layouts/index.html (hunk 2 above)
Comment (robnester-rh) [reply website/layouts/index.html:77]: Yeah, I think this might be the biggest factor for creating a separate page. I'll follow up with more.

**Pull request merged**